optuna_dashboard/ts/components/GraphContour.tsx

```diff
@@ -47,9 +49,18 @@ export const Contour: FC<{
     setYParam(param || null)
   }
 
+  const colorTheme = useRecoilValue<PlotlyColorTheme>(plotlyColorTheme)
+
   useEffect(() => {
     if (study != null) {
-      plotContour(study, objectiveId, xParam, yParam, theme.palette.mode)
+      plotContour(
+        study,
+        objectiveId,
+        xParam,
+        yParam,
+        theme.palette.mode,
+        colorTheme
+      )
     }
   }, [study, objectiveId, xParam, yParam, theme.palette.mode])
```

Comment:
You need to add `colorTheme` for all useEffect, otherwise the change will not be effected immediately.